src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -48,8 +48,6 @@ $spacing: calc(1 * min(var(--section-seat-size), var(--section-seat-size-min)));
     width: 100%;
     height: 100%;
     border: thin $black;
-    border-radius: $border-radius--lg;
-    box-shadow: $iframe-box-shadow;
```

Comment:
Why were these removed from here? I believe the intent was for all the changes to be made within `src/scss/customOverrides.scss` was it not?

Reply:
Oh I thought this was just for auditorium v1 changes. I'll put that in customOverrides then

Reply:
Those styles were removed from `AllSectionPreviews.scss` which is the other auditorium v2 component, so I thought that I should follow that. I'll add both back to their files and then 'remove' them in customOverrides

Reply:
added all overriding styles to customOverrides in c129a927c7a43aec64d1228a2bf25f6194528f40